src/_alfasim_sdk/alfacase/generate_case_description_docstring.py

```diff
@@ -60,21 +61,48 @@ def generate_definition(class_name: str) -> str:
 def _generate_declaration_for_class(class_: Any) -> List[str]:
     """ Return all attributes for the given Class with CaseDescription definition. """
     class_fields = attr.fields_dict(class_)
-    lines = [f"{INDENT*2}class {class_.__name__}"]
-    lines.extend(_get_declaration(class_fields, LIST_OF_CASE_ATTRIBUTES))
-    return lines
+    return [
+        f"{INDENT*2}class {class_.__name__}",
+        *_get_declaration(class_fields, LIST_OF_CASE_ATTRIBUTES),
+    ]
 
 
 def _generate_declaration_for_schema(class_: Any) -> List[str]:
     """ Return all attributes for the given Class using ALFACase schema definition. """
     class_fields = attr.fields_dict(class_)
-    lines = _get_declaration(class_fields, LIST_OF_CASE_SCHEMAS)
-    return lines
+    return _get_declaration(class_fields, LIST_OF_CASE_SCHEMAS, is_schema=True)
+
+
+def _get_default_value_of_case(value: attr.Attribute) -> str:
```

Comment:
This name is a bit strange, how about `_get_case_attr_default_value`?

Reply:
Sure